DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -183,10 +192,10 @@ public void VerifyAdminUserCanAccessCourse_should_return_false_with_incorrect_ce
         {
             // Given
             A.CallTo(() => courseDataService.GetCourseValidationDetails(A<int>._))
-                .Returns((2, 2));
+                .Returns((2, 2, false));
 
             // When
-            var result = courseService.VerifyAdminUserCanAccessCourse(1, 1, 2);
+            var result = courseService.VerifyAdminUserCanManageCourse(1, 1, 2);

```

Comment:
Test name `VerifyAdminUserCanAccessCourse_should_return_false_with_incorrect_centre`
should change to match the new target of VerifyAdminUserCanManageCourse:
`VerifyAdminUserCanManageCourse_should_return_false_with_incorrect_centre`